test/Main.purs

```diff
@@ -81,6 +81,11 @@ main = do
   log "Parsing/printing"
   log ""
 
+  log "An empty string parses Nothing"
+  quickCheck
```

Comment:
Probably better to use `quickCheck' 1` here, because this only needs to be tested once.